src/main/java/calculation/InputView.java

```diff
@@ -0,0 +1,28 @@
+package calculation;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static Scanner scanner = new Scanner(System.in);
```

Comment:
소소한 의견이긴 한데 이 부분도 정적 상수가 되면 좋을 것 같네요 😄 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```